readthedocs/oauth/admin.py

```diff
@@ -16,17 +16,26 @@ class RemoteRepositoryAdmin(admin.ModelAdmin):
 
     """Admin configuration for the RemoteRepository model."""
 
-    raw_id_fields = ('users',)
+    raw_id_fields = ('project', 'organization',)
 
 
-class RemoteOrganizationAdmin(admin.ModelAdmin):
+class RemoteRepositoryRelationAdmin(admin.ModelAdmin):
 
-    """Admin configuration for the RemoteOrganization model."""
+    """Admin configuration for the RemoteRepositoryRelation model."""
 
-    raw_id_fields = ('users',)
+    raw_id_fields = ('account', 'remote_repository', 'user',)
+    list_select_related = ('remote_repository', 'user',)
+
+
+class RemoteOrganizationRelationAdmin(admin.ModelAdmin):
+
+    """Admin configuration for the RemoteOrganizationRelation model."""
+
+    raw_id_fields = ('account', 'remote_organization', 'user',)
+    list_select_related = ('remote_organization', 'user',)
 
 
 admin.site.register(RemoteRepository, RemoteRepositoryAdmin)
-admin.site.register(RemoteRepositoryRelation)
-admin.site.register(RemoteOrganization, RemoteOrganizationAdmin)
```

Comment:
Removed `RemoteOrganizationAdmin` because the only attribute it modifies (`raw_id_fields = ('users',)`) is no longer required because of the `through` model.